travis/install.sh

```diff
@@ -81,7 +81,7 @@ if [ $TRAVIS_OS_NAME == "osx" ]; then
 make -j3
 make check
 PROJ_LIB=$GRIDDIR ./src/gie ./test/gie/builtins.gie
-PROJ_LIB=$GRIDDIR ./src/gie ./test/gie/more_builtins.gie
+PROJ_LIB=$GRIDDIR ./src/gie -vvvvv ./test/gie/more_builtins.gie
```

Comment:
Same comment as in the appveyor file. Why the extra need for verbosity here?

Reply:
Same reply :-)